index.js

```diff
@@ -0,0 +1,172 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex');
+const knexConfig = require('./knexfile');
+const db = knex(knexConfig.development)
+const server = express();
+
+server.use(helmet());
+server.use(express.json());
+
+// endpoints here
+server.get('/api/cohorts', (req, res) => {
+  db.select().from('cohorts')
+    .then(cohorts => res.status(200).json(cohorts))
+    .catch(err => res.status(500).json(err))
+})
+
+server.post('/api/cohorts', (req, res) => {
+  db.select().from('cohorts')
```

Comment:
Currently this request handler functions like a GET request
```suggestion
  db("cohorts").insert(req.body)
```